pkg/resources/resource_grant_cluster_test.go

```diff
@@ -51,6 +51,42 @@ func TestResourceGrantClusterCreate(t *testing.T) {
 	})
 }
 
+func TestResourceGrantClusterCreateP(t *testing.T) {
+	utils.SetDefaultRegion("aws/us-east-1")
+	r := require.New(t)
+
+	in := map[string]interface{}{
+		"role_name":    "PUBLIC",
+		"privilege":    "CREATE",
+		"cluster_name": "materialize",
+	}
+	d := schema.TestResourceDataRaw(t, GrantCluster().Schema, in)
+	r.NotNil(d)
+
+	testhelpers.WithMockProviderMeta(t, func(db *utils.ProviderMeta, mock sqlmock.Sqlmock) {
+		// Create
+		mock.ExpectExec(
+			`GRANT CREATE ON CLUSTER "materialize" TO PUBLIC;`,
+		).WillReturnResult(sqlmock.NewResult(1, 1))
+
+		// Query Grant Id
+		gp := `WHERE mz_clusters.name = 'materialize'`
+		testhelpers.MockClusterScan(mock, gp)
+
+		// Query Params
+		pp := `WHERE mz_clusters.id = 'u1'`
+		testhelpers.MockClusterScan(mock, pp)
+
+		if err := grantClusterCreate(context.TODO(), d, db); err != nil {
```

Comment:
Out of curiosity, what does `context.TODO()` do here?

Reply:
Thanks Jun 🙌 

The `context.TODO()` returns a non-nil, empty Context. It is just used as a placeholder.